src/engine.ts

```diff
@@ -1274,6 +1274,23 @@ export default class Engine {
     this.player(player).placeShip(hex);
   }
 
+  _move(player: PlayerEnum, ship: string, dest: string) {
```

Comment:
Why not a [Command.MoveShip]?

Reply:
:) Solved later

Reply:
The reason is that there is already a Engine.move command, that would conflict with this command. Hence why I add to do something like [`_${Command.MoveShip]]`. We could also replace all the commands by [`command${Command.XXX}`] and phases by [`phase${Phase.XXX}`] to make it more generic!